source/install/requirements.rst

```diff
@@ -101,60 +101,31 @@ Hardware Sizing for Team Deployments
 
 Most small to medium Mattermost team deployments can be supported on a single server with the following specifications based on registered users:
 
--  250-500 users - 2 vCPUs/cores, 4 GB RAM, and 45-90 GB storage
--  500-1,000 users - 4 vCPUs/cores, 8 GB RAM, and 90-180 GB storage
--  1,000-2,000 users - 4-8 vCPUs/cores, 16-32 GB RAM, and 180-360 GB storage
+-  1 - 1,000 users - 1 vCPU/cores, 2 GB RAM, and 45-90 GB storage
```

Comment:
I'm not sure if the storage recommendations are relevant, but we have a note `Storage recommendation is based on storing 3 years of archives with moderate file sharing.`.

Wondering if we should instead ask the admin to estimate file storage requirements based on file sharing within their team. I feel it's highly dependent on the use cases (e.g. a design firm probably requires a lot more file storage capacity compared to a dev ops team)

Reply:
0/5 the storage will also only matter if they're not using minio or S3